src/script/components/asset/LocationAsset.test.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Wire
+ * Copyright (C) 2021 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import TestPage from 'Util/test/TestPage';
+
+import LocationAsset, {LocationAssetProps} from './LocationAsset';
+
+import type {Location} from 'src/script/entity/message/Location';
+
+class LocationAssetPage extends TestPage<LocationAssetProps> {
+  constructor(props?: LocationAssetProps) {
+    super(LocationAsset, props);
+  }
+
+  getMapsElement = () => this.get('[data-uie-name="location-asset-link"]');
+  getMapsLink = () => this.getMapsElement().getDOMNode<HTMLLinkElement>().href;
```

Comment:
`this.getMapsElement().props().href` should do as well without going down to the node.